src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/Deploy.kt

```diff
@@ -108,6 +113,8 @@ open class Deploy : Sync() {
 
     @TaskAction
     open fun deploy() {
+        tailTaskNotification()
```

Comment:
I was more thinking to pin it as it is done to log GAP version

see `AemPlugin.once`

Reply:
I solved it in a different way - showing once every 24h